linodecli/plugins/obj.py

```diff
@@ -667,11 +686,39 @@ def call(args, context):
     if not access_key:
         access_key, secret_key = _get_s3_creds(context.client)
 
-    client = _get_boto_client(parsed.cluster or 'us-east-1', access_key, secret_key)
+    cluster = parsed.cluster
+    if context.client.defaults:
+        cluster = cluster or context.client.config.plugin_get_value('cluster')
+
+    def get_client():
+        """
+        Get the boto client based on the cluster, or ask to configure a default cluster if one is not specified.
+        This is in a method so command methods can do this work AFTER displaying help, that way help can be shown
+        without specifying a cluster or having a valid OBJ key.
+        """
+        current_cluster = cluster
+        if current_cluster is None and not context.client.defaults:
+            print("Error: cluster is required.")
+            exit(1)
+
+        if current_cluster is None:
+            print("Error: No default cluster is configured.")
+            print("")
+
+            answer = input_helper("Configure now? [y/N] ")
```

Comment:
I think you should remove this prompt and just dive into configuring the plugin if necessary - it doesn't work if you say no, and if you don't to pick a cluster you can just `^C`.  This is how the generated CLI handles configuration.